tests/TestUser.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Konekt\AppShell\Tests;
+
+use Konekt\AppShell\Models\User;
+
+class TestUser extends User
+{
+    protected $guard_name = 'web';
```

Comment:
This looks like faking the result, as there's no such requirement that user should have this property defined.
We need a solution without this. If you want it merged, please at least mark it as todo, push it to this branch and fix this hack in your next PR.